lib/components/narrative/default/default-itinerary.js

```diff
@@ -123,9 +123,7 @@ const ITINERARY_ATTRIBUTES = [
       const fareCurrency = itinerary.fare?.fare?.regular?.currency?.currencyCode
       const fare = fareInCents === null ? null : fareInCents / 100
```

Comment:
I suppose this logic could be reused in the trip-details component to handle cases where cents show as `-1`. (If that's the case, trip details and the itinerary summary render different things).

Reply:
Noted in https://github.com/opentripplanner/otp-ui/issues/576.

Reply:
Good call thanks. Yes I think this should be in core-utils eventually